recipe/meta.yaml

```diff
@@ -102,12 +103,14 @@ outputs:
         - lib/R/lib
     requirements:
       build:
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - git
-        - cmake
-        - make
-        - llvm-openmp  # [osx]
+        - {{ compiler('c') }}    # [not win]
+        - {{ compiler('cxx') }}  # [not win]
+        - llvm-openmp            # [osx]
+        - m2w64-toolchain        # [win]
+        - git                    # [not win]
+        - m2-git                 # [win]
```

Comment:
I am not sure if this makes a difference, but I have seen `git` as build requirement in outputs defined above, but no `m2-git`. The missing, or the additional git might cause the problems?

Reply:
`git` is broken on windows. `m2-git` is preferred.

Reply:
Do we have an issue to track that? I'm using `git` on Windows from time to time :D

Reply:
Nope.  `git` package on windows is bundling its on msys2 packages and when conda msys2 packages are active, they mess up.